llvm/lib/Target/SPIRV/SPIRVOperands.h

```diff
@@ -0,0 +1,204 @@
+//===-- SPIRVOperands.h -  Top level definitions for SPIRV ------*- C++ -*-===//
```

Comment:
IMO this file name is misleading, I'd suggest something like 'SPIRVSymbolicOperands' or something like that